base/set.jl

```diff
@@ -645,6 +645,8 @@ replace!(prednew::Callable, A; count::Integer=typemax(Int)) =
     replace!(prednew, A, count=clamp(count, typemin(Int), typemax(Int)) % Int)
 
 
+replace!(prednew::Callable, A; count::Int=typemax(Int)) =
+throw(MethodError(replace!, (prednew, A, count)))
```

Comment:
Missing indentation. Also, `count` is not a positional argument, so I don't think it should be passed to `MethodError` (I'm not sure how it could be passed).

Reply:
The first commit was by error - I will back that out.
Next commit will include your suggestions.